week1/1.py

```diff
@@ -0,0 +1,4 @@
+name  = input("enter name ")
+age = int(input("enter age "))
+
+print("Your name: "+name+" and age: "+str(age))
```

Comment:
Just like in JS there are template string, in Python there are f-strings. Google for them and see how you can improve your print statement